aws/table_aws_route53_record.go

```diff
@@ -20,10 +20,11 @@ func tableAwsRoute53Record(_ context.Context) *plugin.Table {
 		Name:        "aws_route53_record",
 		Description: "AWS Route53 Record",
 		List: &plugin.ListConfig{
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"InvalidInput"}),
+			},
 			KeyColumns: []*plugin.KeyColumn{
```

Comment:
```suggestion
 			// Some optional key columns are disabled due to https://github.com/aws/aws-sdk-go/issues/4386
			KeyColumns: []*plugin.KeyColumn{
```

@bigdatasourav can you please add this comment, comment out the lines below instead of removing them, and similarly comment out the code below so we can re-add it easily once the bug is fixed?

Reply:
Also, can you please create a new issue to re-enable these key columns once the SDK issue is resolved?